app/views/shared/record/_record_toolbar_tools_subset.html.erb

```diff
@@ -18,7 +18,11 @@
 <% end %>
 <% if @document.export_formats.keys.include?( :refworks_marc_txt ) %>
   <li class="refworks" role="presentation">
-    <%= link_to t('blacklight.tools.refworks_html'), refworks_export_url({url: polymorphic_url(url_for_document(@document), format: :refworks_marc_txt, only_path: false)})  %>
+    <%= form_tag(refworks_export_url) do %>
+      <% # TODO: Remove send when Blacklight::Marc exposes this as a public method %>
```

Comment:
```suggestion
      <% # TODO: Remove send when Blacklight::Marc exposes this as a public method projectblacklight/blacklight-marc#64 %>
```
^-- thought it might be nice to throw the issue number in here in case the upstream work doesn't happen by the end of this work cycle